README.md

```diff
@@ -1 +1,136 @@
-# Hello
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-microverse] <a name="Hello-microverse"></a>
+
+**[Hello-microverse]** this was mini-projects that upskill to create a feature branch and test linters in order to have professional-looking code.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your first "Hello-microverse" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+<br>
+-Node JS
+<br>
+-Visual Studio Code or any other code editor of your preference
+<br>
+-web browser
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+git@github.com:brighton0102/Hello.git
+
+## Install Linters
+you can use 
+
+Github
+VS code
+so that you can access the code
+
+To run the project, execute the following commands
+
+Linters
+git commands
+
```

Comment:
- [ ] Please always add the relevant information in your `README.md` file because `README.md` is just like a guide for someone who is going to use the project and makes the person understand how your project would work. As mentioned by your previous code reviewer don't need to tell the whole process instead you can add something similar to below given suggestion.

```
## Install Linters

npm i
```

- Above example will help the user to simplify the linter installation process rather put a user into confusion.